chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/certifiedProduct/AmbulatoryModular2014Validator.java

```diff
@@ -91,8 +93,25 @@ protected void validateDemographics(PendingCertifiedProductDTO product) {
 							product.getErrorMessages().add("There was no test tool found matching '" + testTool.getName() + "' for certification " + cert.getNumber() + ".");
 						} else {
 							TestToolDTO tt = ttDao.getById(testTool.getTestToolId());
-							if(tt != null && tt.isRetired()) {
-								product.getErrorMessages().add("Cannot use test tool '" + tt.getName() + "' since it is retired. Please choose a different test tool for " + cert.getNumber() + ".");
+							String[] idParts = product.getUniqueId().split("\\.");
+							if(idParts.length < 9) {
+								logger.error("CHPL ID must have 9 parts separated by '.'");
+							}
+							if(tt != null && tt.isRetired() && idParts[6].toString().equals("0")) {
+								Boolean containsInheritedCertificationStatus = false;
+								for (String error: product.getErrorMessages()) {
+									if(error.contains("Inherited Certification Status is true")){

```

Comment:
I don't think it's best practice to compare on this string if we decide to change the error message in the future.  A better way might be to set a protected field in the parent class where the error about ICS conflict is originally generated and just check that. Then you won't have to worry about parsing the ID into its separate parts either.